SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -154,6 +154,13 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ToolTip = "Check is the Actor has it's authority delegated to a specific Server Worker. \n\nKeep in mind that currently this functionality only works in single layer Load Balancing Strategies, and your Default Load Balancing Strategy needs to implement ISpatialFunctionalTestLBDelegationInterface."))
 	bool HasActorDelegation(AActor* Actor, int& WorkerId, bool& bIsPersistent);
 
+	// # Actor Interest APIs
+	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1", ToolTip = "Allow Server Worker to always have interest in an Actor, even it if moves outside its interest area.\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
+	void AddActorInterest(int32 ServerWorkerId, AActor* Actor);
+
+	UFUNCTION(BlueprintCallable, Category = "Spatial Functional Test", meta = (ServerWorkerId = "1", ToolTip = "Counterpart to AddActorInterest().\n\nNote that this only works if you are using USpatialFunctionalTestGridLBStrategy."))
```

Comment:
Super nit: I would maybe mention that this explicitly removes the interest above - you can't use this to become uninterested in an actor, which has interest through the traditional GDK means.
Not essential, though.